src/app/gui/queryresults/queryresultsservice.js

```diff
@@ -1,69 +1,205 @@
+import { G3W_FID, LIST_OF_RELATIONS_TITLE, LIST_OF_RELATIONS_ID } from 'constant';
 import ProjectsRegistry from 'store/projects';
 import DataRouterService from 'services/data';
 import CatalogLayersStoresRegistry from 'store/catalog-layers';
 import DownloadFormats from 'components/QueryResultsActionDownloadFormats.vue';
 import QueryPolygonCsvAttributesComponent from 'components/QueryResultsActionQueryPolygonCSVAttributes.vue';
 import ApplicationService from 'services/application';
 
-const {base, inherit, noop, downloadFile, throttle, getUniqueDomId, copyUrl } = require('core/utils/utils');
 const {
+  base,
+  inherit,
+  noop,
+  downloadFile,
+  throttle,
+  getUniqueDomId,
+  copyUrl,
+} = require('core/utils/utils');
+
+const {
+  getAlphanumericPropertiesFromFeature,
   createFeatureFromFeatureObject,
+  createFeatureFromGeometry,
+  createFeatureFromBBOX,
+  createFeatureFromCoordinates,
   intersects,
-  within
+  within,
 } = require('core/utils/geo');
-const {getAlphanumericPropertiesFromFeature, createFeatureFromGeometry, createFeatureFromBBOX, createFeatureFromCoordinates} = require('core/utils/geo');
-const {t} = require('core/i18n/i18n.service');
-const Layer = require('core/layers/layer');
-const G3WObject = require('core/g3wobject');
-const VectorLayer = require('core/layers/vectorlayer');
-const PrintService = require('core/print/printservice');
-const RelationsPage = require('gui/relations/vue/relationspage');
+
+const { t }                      = require('core/i18n/i18n.service');
+const Layer                      = require('core/layers/layer');
+const G3WObject                  = require('core/g3wobject');
+const VectorLayer                = require('core/layers/vectorlayer');
+const PrintService               = require('core/print/printservice');
+const RelationsPage              = require('gui/relations/vue/relationspage');
 const PickCoordinatesInteraction = require('g3w-ol/interactions/pickcoordinatesinteraction');
 
-//used to get and set vue reactivity to queryresultservice
+const deprecate = require('util-deprecate');
+
+/**
+ * Get and set vue reactivity to QueryResultsService
+ * 
+ * @type {Vue}
+ */
 const VM = new Vue();
 
 function QueryResultsService() {
+
+  /**
+   * Service used to work with atlas (print functionality) action tool
+   */
   this.printService = new PrintService();
+
+  /**
+   * @FIXME add description
+   */
   this._currentLayerIds = [];
 
+  /**
+   * @deprecated since 3.8
+   * It used to register change project from Change map button
+   */
   ProjectsRegistry.onafter('setCurrentProject', project => {
     this._project = project;
     this._setRelations(project);
     this._setAtlasActions(project);
     this.state.download_data = false;
     this.plotLayerIds = [];
   });
+
+  /**
+   * @FIXME add description
+   */
   this.unlistenerlayeractionevents = [];
+
+  /**
+   * @FIXME add description
+   */
   this._actions = {
     'zoomto': QueryResultsService.zoomToElement,
```

Comment:
```js
QueryResultsService.zoomToElement
```

is this an undefined function?

Reply:
@Raruto yes. You can delete it. Thanks

Reply:
removed in https://github.com/g3w-suite/g3w-client/pull/317/commits/40d0254fb7a5d2bf20e59990c20cde13f92694df